README.md

```diff
@@ -27,14 +27,14 @@ See the official [React Native website](https://facebook.github.io/react-native/
 
 ## System Requirements
 
-- You can build and deploy React Native Windows apps on the Pro or Enterprise versions of Windows 7 SP1, Windows 8.1, or Windows 10 
+- You can build and deploy React Native Windows apps on the Pro or Enterprise versions of Windows 7 SP1, Windows 8.1, or Windows 10
 - *Note:* You cannot run the emulators and some other developer tooling on the "Starter" or "Home" versions of these operating systems
 - You can run React Native Windows UWP apps only on Windows 10 devices, but React Native Windows WPF apps will run on Windows 7-10 so long as .NET 4.6 is installed on the end user's machine
-- Download [Visual Studio 2015 Community](https://www.visualstudio.com/vs/older-downloads/)
+- Download [Visual Studio 2015 Community or Greater](https://www.visualstudio.com/vs/older-downloads/) or [Visual Studio 2017 Community or Greater](https://www.visualstudio.com/downloads/)
```

Comment:
@sayar can you change this line to "Download Visual Studio 2017 or greater"? You can add "Visual Studio 2015 support is being deprecated" as well.